soccer/gameplay/tactics/test_cost_function.py

```diff
@@ -0,0 +1,9 @@
+import robocup
+import constants
+import math
+
+## A test cost function for passes that should have a minima at (0.915239, 1.38493)
+def testCostFunction(coordinates) -> float:
```

Comment:
If the heuristic wants extra args, is the plan to bind them similar to std::bind leaving the coordinates as the only arg? The other option would be a tuple of args that get forwarded into the minimize function, and is passed directly into this every call